integration/kubernetes/k8s-volume.bats

```diff
@@ -25,6 +25,8 @@ setup() {
 }
 
 @test "Create Persistent Volume" {
+	wait_time=10
+	sleep_time=3
```

Comment:
I'd prefer a small wait time and more tries, how about we use 5 and 6 ?

Reply:
ok, changed to try every 2s.